src/main/java/com/example/sixthmafiabot/services/GameService.java

```diff
@@ -13,14 +13,13 @@
 import org.springframework.stereotype.Service;
 
 import java.util.concurrent.CompletableFuture;
-import java.util.concurrent.ExecutionException;
 
 @Service
 @Slf4j
 public class GameService implements BaseService {
 
     @Autowired
-    GameRepository gameRepository;
+    SpringGameRepository springGameRepository;
```

Comment:
you must use GameRepo instead of SpringGameRepo

Reply:
+